sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -136,6 +136,7 @@ void gnrc_ipv6_nib_iface_up(gnrc_netif_t *netif)
     netif->ipv6.rs_sent = 0;
 #endif  /* CONFIG_GNRC_IPV6_NIB_6LN */
     netif->ipv6.na_sent = 0;
+    _auto_configure_addr(netif, &ipv6_addr_link_local_prefix, 64U);
```

Comment:
If we do this in `gnrc_ipv6_nib_iface_up()` we can remove it from `gnrc_ipv6_nib_init_iface()`

Reply:
Done, but not tested so far.